source/index.html.md.erb

```diff
@@ -97,11 +97,13 @@ GDS Metrics is in alpha. These instructions are subject to change.
 
 Several teams in GDS use [Amazon Web Services (AWS)][] as their infrastructure provider.
 
-GDS teams manage their own AWS accounts, but users must sign into a shared base AWS account managed by Reliability Engineering. You can find out:
+GDS teams can create and manage their own AWS accounts, but all users must first sign into a shared base AWS account managed by Reliability Engineering.
 
-- [how to access AWS accounts][]
-- [how to create AWS accounts][]
-- [how AWS cross account access works][]
+Find out how to:
+
+- [access AWS][]
+- [request AWS accounts][]
+- [manage AWS cross account account access][]
```

Comment:
This link is broken - needs to match the one on Line 125

Reply:
any reason to not put the links directly there with []()?

Reply:
`[]()`

Reply:
https://github.com/alphagov/reliability-engineering/pull/22#discussion_r190833905

Reply:
corrected that now.
@willp-bl it's just to keep the copy clear, in some of our guidance the links can be pretty long, I find it easier to edit if the URLs are out of the way.